src/middleware/with-webhook-signature-verified.ts

```diff
@@ -6,14 +6,21 @@ import { Response } from "retes/response";
 import { SALEOR_SIGNATURE_HEADER } from "../const";
 import { getSaleorHeaders } from "../headers";
 import { getJwksUrl } from "../urls";
+import { createMiddlewareDebug } from "./middleware-debug";
+
+const debug = createMiddlewareDebug("withWebhookSignatureVerified");
+
+const ERROR_MESSAGE = "Webhook signature verification failed:";
 
 export const withWebhookSignatureVerified =
   (secretKey: string | undefined = undefined): Middleware =>
   (handler) =>
   async (request) => {
-    const ERROR_MESSAGE = "Webhook signature verification failed:";
+    debug("Middleware executing start");
 
     if (request.rawBody === undefined) {
+      debug("Request rawBody was not found, will return Internal Server Error");
```

Comment:
```suggestion
      debug("Request rawBody was not found");
```

Don't we need to additionally describe what middleware will return - describing everything will get tedious very fast IMO.

Reply:
I'm not sure. Response from middleware isn't traced anywhere right? Retrning handler is easy to debug because next messages will follow. Response is breaking the chain, and this error can be found in frontend. IMO it "closes" debug flow and you know that this response is over. 

Maybe we can implement a DebugMiddleware that will just optionally log responses?

Reply:
Like the idea of debugging middleware - it could log Request and Responses 👍

Reply:
added experimental middleware